README.md

```diff
@@ -0,0 +1,7 @@
+In this project, I worked on a portfolio mobile version project and created two extra sections to add up to the original section. The sections created for the current project were: "works section" to showcase my previous projects I have worked on and the "about myself section" to showcase my programming experiences.
+During the project, I have learned the following:
+
+How to create cards
+How to use Flexbox to position and align elements
+How to use Grid layout to position elements
+How to create cards and style them
```

Comment:
- [ ] Kindly use the microverse readme template that can be found here [microverse readme template](https://github.com/microverseinc/readme-template)